utils/wasm-instrument/src/syscalls.rs

```diff
@@ -193,6 +193,8 @@ impl SysCallName {
             Self::Read,
             Self::Reply,
             Self::ReplyWGas,
+            Self::ReplyInput,
```

Comment:
how could it be wout failing on the backend side?

Reply:
we have a validation on amount of declared syscalls there

Reply:
The answer is that the method is used only in one test and in wasm-gen.